server/config/default.js

```diff
@@ -23,5 +23,6 @@ module.exports = {
     secret: process.env.JWT_SECRET || 'addadasjfghKEU10dawadrfgh!e29sosafelol',
     expiry: process.env.JWT_EXPIRY || '30m',
   },
+  mail: 'ERP_Soen390-team07',
```

Comment:
if this is the password, I would'nt expose this to github since this is a legit email account

Reply:
I was thinking the same, I am looking for a solution for that. Any suggestions?

Reply:
```suggestion
  mail: 'process.env.EMAIL_PASSWORD',
```

Then create a .env file and store the password there.
EMAIL_PASSWORD='ERP_Soen390-team07'